src/include/timestamp.i

```diff
@@ -0,0 +1,196 @@
+/*-
+ * Copyright (c) 2014-2020 MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *	All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+/*
+ * __wt_time_window_init --
+ *     Initialize the fields in a time window to their defaults.
+ */
+static inline void
+__wt_time_window_init(WT_TIME_WINDOW *tw)
+{
+    tw->start_durable_ts = WT_TS_NONE;
+    tw->start_ts = WT_TS_NONE;
+    tw->start_txn = WT_TXN_NONE;
+    tw->stop_durable_ts = WT_TS_NONE;
+    tw->stop_ts = WT_TS_MAX;
+    tw->stop_txn = WT_TXN_MAX;
+    tw->prepare = false;
+}
+
+/*
+ * __wt_time_window_init_max --
+ *     Initialize the fields in a time window to values that force an override.
+ */
+static inline void
+__wt_time_window_init_max(WT_TIME_WINDOW *tw)
+{
+    tw->start_durable_ts = WT_TS_MAX;
+    tw->start_ts = WT_TS_MAX;
+    tw->start_txn = WT_TXN_MAX;
+    tw->stop_durable_ts = WT_TS_MAX;
+    tw->stop_ts = WT_TS_NONE;
+    tw->stop_txn = WT_TXN_NONE;
+    tw->prepare = false;
+}
```

Comment:
These two init functions could be combined to just one and take an arg to indicate how to initialize.

Reply:
The "initialize to max" is a really limited function, it's only used by reconciliation, so I'm inclined to leave this as it is. Let me know if you feel strongly.